internal/bminventory/inventory_test.go

```diff
@@ -11548,7 +11659,7 @@ var _ = Describe("TestRegisterCluster", func() {
 				reply := bm.V2RegisterCluster(authCtx, installer.V2RegisterClusterParams{
 					NewClusterParams: &models.ClusterCreateParams{
 						Name:             swag.String("some-cluster-name"),
-						OpenshiftVersion: swag.String("some-version-doesnt-matter-because-we-mock"),
+						OpenshiftVersion: swag.String("4.12.0-someFakeFlavour"),
```

Comment:
Can we maybe have a global test var called `var arbitraryOCPVersion := swag.String("4.12.0-someFakeFlavour")`? for all those tests you modified that don't particularly care about the OpenShift version but must specify it because it's required. It reduces mental overhead when reading those tests, avoiding questions like "Why are they using 4.12.0 specifically here?" etc